.changeset/45-update-eslint-plugin-import.md

```diff
@@ -0,0 +1,6 @@
+---
+issue: https://github.com/pmedianetwork/eslint-config/issues/45
+type: minor
```

Comment:
I think we also have to actually set the rule:

```
"import/no-cycle": "error"
```

It is not set at all at the moment. Currently, we only have it in `insights`, and there, we set it to "warn".

_IF_ we do this, it's most likely a breaking change. Do we want that @danielbartsch ❓ 
I am also not sure how this will integrate with typescript ... But we could just say that repos that don't want it that strict need to "downgrade" the rule "warn" for themselves .. ?

Reply:
It is a good rule to have. 
If one some of the projects have issues with it, as last resort they can disable it with `"import/no-cycle": 0`. That could be a good opportunity to check the health of the codebase though.

Reply:
since this https://github.com/benmosher/eslint-plugin-import/pull/1494 is merged, I'd do it, even tho yes, it is a breaking change.

But since we have 2 other PRs merged, I'd do this after the release.

Reply:
okay, I'll make a minor release and then we can maybe do this as a breaking release, maybe together with #39